src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -1211,20 +1210,28 @@ NVTX.@annotate function update_sw_fluxes!(
     )
 end
 
-# TODO: Change these methods back to using face_sw_flux/face_clear_sw_flux
-# instead of face_sw_flux_up - face_sw_flux_dn
-update_net_fluxes!(radiation_mode, model) =
-    parent(model.face_flux) .=
-        parent(model.face_lw_flux) .+ parent(model.face_sw_flux_up) .-
-        parent(model.face_sw_flux_dn)
+function update_net_fluxes!(_, model)
+    model.face_flux .=
+        ifelse.(
+            model.cos_zenith' .≈ 0.0,
```

Comment:
Could we add the tolerance explicitly so we know what tolerance we are using?

Reply:
The kwarg `rtol = ...` seems to cause GPU compilation errors, so I have left it as a comment - see [here](https://buildkite.com/clima/climaatmos-ci/builds/16873#018dd79c-0fa3-461a-8400-ea130d58e917)

Reply:
Could we just do `model.cos_zenith' <= sqrt(eps(FT))` here? Or this wouldn't work on GPU either?

Reply:
That should work - i will change it!